### PR TITLE
terminal.vala: Use the cursor shape from settings

### PR DESCRIPTION
**Pull request by aleksandar-stefanovic**
Does what it says on the tin

### CODE REVIEW

File: plugins/terminal/terminal.vala [hunk 1]
```diff
@@ -328,6 +328,14 @@ public class Scratch.Plugins.Terminal : Peas.ExtensionBase,  Peas.Activatable {
         bool audible_bell_setting = pantheon_terminal_settings.get_boolean ("audible-bell");
         this.terminal.set_audible_bell (audible_bell_setting);
 
+        string cursor_shape_setting = pantheon_terminal_settings.get_string("cursor-shape");
+
+        switch(cursor_shape_setting) {
+            case "Block": this.terminal.cursor_shape = CursorShape.BLOCK; break;
```
Comment (donadigo) [inline plugins/terminal/terminal.vala]: Split this line two lines:
```
case "Block":
    this.terminal.cursor_shape = CursorShape.BLOCK;
    break;
case "I-Beam":
...
```

File: plugins/terminal/terminal.vala [hunk 2]
```diff
@@ -328,6 +328,14 @@ public class Scratch.Plugins.Terminal : Peas.ExtensionBase,  Peas.Activatable {
...
+            case "I-Beam": this.terminal.cursor_shape = CursorShape.IBEAM; break;
```
Comment (donadigo) [inline plugins/terminal/terminal.vala]: Same here.

File: plugins/terminal/terminal.vala [hunk 3]
```diff
@@ -328,6 +328,14 @@ public class Scratch.Plugins.Terminal : Peas.ExtensionBase,  Peas.Activatable {
...
+            case "Underline": this.terminal.cursor_shape = CursorShape.UNDERLINE; break;
```
Comment (donadigo) [inline plugins/terminal/terminal.vala]: Same here.

File: plugins/terminal/terminal.vala [hunk 4]
```diff
@@ -328,6 +328,14 @@ public class Scratch.Plugins.Terminal : Peas.ExtensionBase,  Peas.Activatable {
         bool audible_bell_setting = pantheon_terminal_settings.get_boolean ("audible-bell");
         this.terminal.set_audible_bell (audible_bell_setting);
 
+        string cursor_shape_setting = pantheon_terminal_settings.get_string("cursor-shape");
```
Comment (donadigo) [inline plugins/terminal/terminal.vala]: Use space before brackets: `get_string ("cursor-shape")`.

File: plugins/terminal/terminal.vala [hunk 5]
```diff
@@ -328,6 +328,14 @@ public class Scratch.Plugins.Terminal : Peas.ExtensionBase,  Peas.Activatable {
...
+
+        switch(cursor_shape_setting) {
```
Comment (donadigo) [inline plugins/terminal/terminal.vala]: Use space before brackets here too: `switch (cursor_shape_setting) {`

**Comment by aleksandar-stefanovic** [comment]
All okay now?

**Pull request merged**